python/cuda_parallel/cuda/parallel/experimental/_cccl_interop.py

```diff
@@ -220,6 +223,30 @@ def get_paths() -> List[str]:
     return paths
 
 
+def _check_compile_result(cubin: bytes):
+    # check compiled code for LDL/STL instructions
+    temp_cubin_file = tempfile.NamedTemporaryFile(delete=False)
+    try:
+        temp_cubin_file.write(cubin)
+        out = subprocess.run(
+            ["nvdisasm", "-gi", temp_cubin_file.name], capture_output=True
+        )
+        if out.returncode != 0:
+            raise RuntimeError("nvdisasm failed")
+        sass = out.stdout.decode("utf-8")
+    finally:
+        os.unlink(temp_cubin_file.name)
+
+    assert "LDL" not in sass, "LDL instruction found in SASS"
+    assert "STL" not in sass, "STL instruction found in SASS"
+
+
+# this global variable controls whether the compile result is checked
+# for LDL/STL instructions. Should be set to `True` for testing only.
+global _check_sass
```

Comment:
Since `global` is not used in function scope, it does not have any effect here. This line can be deleted.

Reply:
Fixed.